pkg/api/handlers/compat/networks.go

```diff
@@ -289,25 +289,40 @@ func CreateNetwork(w http.ResponseWriter, r *http.Request) {
 
 func RemoveNetwork(w http.ResponseWriter, r *http.Request) {
 	runtime := r.Context().Value("runtime").(*libpod.Runtime)
-	config, err := runtime.GetConfig()
-	if err != nil {
-		utils.InternalServerError(w, err)
+	ic := abi.ContainerEngine{Libpod: runtime}
+
+	query := struct {
+		Force bool `schema:"force"`
+	}{
+		// This is where you can override the golang default value for one of fields
+	}
+
+	decoder := r.Context().Value("decoder").(*schema.Decoder)
+	if err := decoder.Decode(&query, r.URL.Query()); err != nil {
+		utils.Error(w, "Something went wrong.", http.StatusBadRequest, errors.Wrapf(err, "failed to parse parameters for %s", r.URL.String()))
```

Comment:
I'm not going to block just because of this... but if you need to resubmit for any reason, could there maybe be a better diagnostic than "Something went wrong"?

Reply:
Fixed, although this message is in all over the code.  I think this is what Docker did when there was an unexpected error.

Reply:
@edsantiago I thoroughly concur and have been dinging those as I encounter them.  It makes me grind my teeth every time I spot it.